tritium-metrics/src/main/java/com/palantir/tritium/metrics/MetricRegistries.java

```diff
@@ -206,6 +209,29 @@ static void registerCache(MetricRegistry registry, Cache<?, ?> cache, String nam
                 .forEach((key, value) -> registerWithReplacement(registry, key, value));
     }
 
+    public static ScheduledExecutorService instrument(
```

Comment:
These return ExecutorService/ScheduledExecutorService so that we can add additional implementations (may make ThreadPoolExecutor specific metrics cleaner to add in the future).

The overload allows us to route invocations to the most correct method based on the executor type at compile time.